.github/workflows/create-quantic-package.yml

```diff
@@ -0,0 +1,47 @@
+name: Create Quantic Package Workflow
+
+env:
+  SFDX_AUTH_JWT_INSTANCE_URL: https://login.salesforce.com
+  SFDX_AUTH_JWT_USERNAME: sfdc.integration.devv2.hub@coveo.com
+  SFDX_AUTH_JWT_KEY: ${{ secrets.SFDX_AUTH_JWT_KEY_DEV }}
+  SFDX_AUTH_JWT_CLIENT_ID: 3MVG9LBJLApeX_PDlEni7JUnKtxHybHKI_12BhenOY_Av5vP6uR_vGdaNRelKFKaltSVpU3WvrRnaLqr6xeiZ
+  PACKAGE_ID: 0Ho6g000000k9g8CAA
+
+on:
+  workflow_dispatch:
+    inputs:
+      shouldPromote:
```

Comment:
According to @tedre191 the automatic promotion of the package is what we want. However, until we've added a comprehensive cypress test run to this process I agree that we shouldn't promote automatically. We could change to default: true later on? Thoughts?

Reply:
I feel I might have missed a few discussions on that topic, but I don't think we should promote the package automatically, at the very least, not yet. As you mentioned, there is almost no test coverage right now.

So, if I understand correctly, the end goal is to have this workflow generate and promote the package automatically without any manual step, right?

Since we don't have any tests covering Quantic, I would suggest the following:

- remove the `shouldPromote` input parameter
- remove the `promote` stuff from the workflow

This way, the workflow would build the package, and once we are confident the package is correct, we can promote it manually.

Once we have the automated tests we need to ensure the package is correct, then we can modify the workflow to promote the package automatically.

Does it make sense to you?

Reply:
Sounds good to me too